test/tests/recognizePDFTest.js

```diff
@@ -1,4 +1,4 @@
-describe.skip("PDF Recognition", function() {
+describe("PDF Recognition", function() {
```

Comment:
Oh, sorry, didn't realize these were currently disabled. These should actually stay disabled until they mock the HTTP services.

Asking you to do this was actually a bit of a tall order, among other things because these tests predate some advances in Zotero testing technology. Let's leave this commit out, and I'll add a test separately that shows the best way to do this.